assets/Product.wxs

```diff
@@ -269,7 +282,8 @@
       <Control Id="AddToPathCheckBox" Type="CheckBox" X="20" Y="60" Width="290" Height="17" Property="ADD_PATH" CheckBoxValue="0" Text="Add $(var.ProductName) to Path Environment Variable"/>
       <Control Id="RegisterManifestCheckBox" Type="CheckBox" X="20" Y="80" Width="290" Height="17" Property="REGISTER_MANIFEST" CheckBoxValue="0" Text="Register Windows Event Logging Manifest"/>
       <Control Id="EnablePSRemotingCheckBox" Type="CheckBox" X="20" Y="100" Width="290" Height="17" Property="ENABLE_PSREMOTING" CheckBoxValue="0" Text="Enable $(var.ProductName) remoting"/>
-      <Control Id="ContextMenuOpenPowerShell" Type="CheckBox" X="20" Y="120" Width="290" Height="17" Property="ADD_EXPLORER_CONTEXT_MENU_OPENPOWERSHELL" CheckBoxValue="0" Text="Add '$(var.ExplorerContextSubMenuDialogText)' context menus to Explorer"/>
+      <Control Id="DisableTelemetry" Type="CheckBox" X="20" Y="120" Width="290" Height="17" Property="DISABLE_TELEMETRY" CheckBoxValue="0" Text="Disable Telemetry"/>
```

Comment:
```suggestion
      <Control Id="DisableTelemetry" Type="CheckBox" X="20" Y="120" Width="290" Height="17" Property="DISABLE_TELEMETRY" CheckBoxValue="0" Text="Disable PowerShell Telemetry"/>
```

Reply:
Can we have a tooltip?

If so, can we say that a reboot may be required for this to take effect in some cases.

Reply:
I'd maybe even add it in parenthesis but first I would like to understand the scenarios where a reboot is required. Already open pwsh shells would obviously need restarting (e.g. when stable is already installed and the user installs preview) but why would one need a reboot? When I did some early manual testing, it wasn't always working as expected (hence why it is still in draft) and I would like to understand why. Is it because the `explorer.exe` process needs restarting (quite drastic, I know, but some installers like e.g. the one of TortoiseGit do that). Is there an API we could call to inform existing processes about the change?

Reply:
No service will every pick up a new environment variable.  Any process that works in the same model will have the same process.  There is a notification API, but I've never seen any process that pays attention to it.

Reply:
Ok, just looked at WiX docs [here](https://wixtoolset.org/documentation/manual/v3/customactions/wixsettingchange.html) and essentially they say that if we use the `Environment` element as you suggested below, it will send a `WM_SETTINGCHANGE` message to propagate the change (which should hopefully be good enough in most cases and I hope at least `explorer.exe` listens to it), except when there is a pending reboot. Let me try use the `Environment` element and do some testing and come back with results. If results are positive I suggest to change the `Text` to `Disable PowerShell Telemetry (might require reboot)` and add a `ToolTip` with more details around when a reboot is required.

Reply:
`Explorer.exe` does listen or alway creates a new environment (I think the second)